azure-pipelines.yml

```diff
@@ -29,8 +29,20 @@ variables:
   # codecov_token: 4dc9e7e2-6b01-4932-a180-847b52b43d35 # Get a new one from https://codecov.io/
   NUGET_PACKAGES: $(Agent.TempDirectory)/.nuget/packages/
 
-jobs:
-- template: azure-pipelines/build.yml
+resources:
```

Comment:
Changes to this file should probably be reverted. This file is used for the pipelines in the `azure-public` AzDO account, where presumably the 1ES PT or MicroBuild templates don't exist. So it would break the OSS-visible PR builds.

Reply:
Confirmed. This change breaks the OSS build: https://dev.azure.com/andrewarnott/OSS/_build/results?buildId=8897&view=results

Reply:
azure-pipelines.yml runs in non-devdiv accounts.
official.yml runs in devdiv accounts.

Up to this point, build.yml was what they shared while providing unique entrypoints so that pipelines in different AzDO accounts could set different variables and parameters in build.yml.
If build.yml must take changes that cannot be compatible with both environments, then I guess let's leave build.yml alone to continue to serve the non-devdiv accounts, and copy what you need from build.yml directly into official.yml and make modifications there.

Reply:
K.  I've just pushed some changes that I hope will work in both worlds.  I'm using the CollectionID that seems to be used to turn on and off the MicroBuild stuff.  It's mostly around the outputs that has changed.  The 1ES template stuff should be ignored for the non-devdiv stuff and stay with the powershell script way of outputting.  But, for the devdiv stuff it should turn on.